### DISCUSSION TITLE
Usage of announce intent to merge

### DISCUSSION THREAD

**Discussion by tcharding** [rust-bitcoin/rust-bitcoin#2005] [Announcements]
I listened to Bitcoin Optech's [commentary](https://bitcoinops.org/en/podcast/2023/08/10/#rust-bitcoin-1945) on our recent one-ack carve-out (#1995). They suggest announcing intent to merge if a PR is a candidate for the carve-out. This seems like a good idea. I have started to do so on PRs after one ack (typically @apoelstra). If you are a maintainer and you would like some other method of notification, like a label or some other github magic, that allows you to follow such PRs please say so.

Examples
- #1978
- #1988

**Comment by apoelstra** [comment]
@tcharding looks good to me!

**Comment by stevenroose** [comment]
Hey how about we use RSS for things? Isn't GitHub supposed to be good with RSS already built-in? Like I think maybe we can get RSS feeds of MR tags, so then we can subscribe to the ones that will be merged soon?

**Reply by tcharding** [reply]
Cool idea, I'll look into it.

**Reply by apoelstra** [reply]
Agreed, if there were a good RSS integration I'd probably actually use a scheme, even a label based one. Cool idea!
